packages/govuk-frontend/src/govuk/common/index.mjs

```diff
@@ -8,26 +8,6 @@
  * @module common/index
  */
 
-/**
- * TODO: Ideally this would be a NodeList.prototype.forEach polyfill
- * This seems to fail in IE8, requires more investigation.
- * See: https://github.com/imagitama/nodelist-foreach-polyfill
- *
- * @deprecated Will be made private in v5.0
- * @template {Node} ElementType
- * @param {NodeListOf<ElementType>} nodes - NodeList from querySelectorAll()
- * @param {(value: ElementType, index: number, nodes: NodeListOf<ElementType>) => void} callback - Callback function to run for each node
- * @returns {void}
- */
-export function nodeListForEach (nodes, callback) {
-  if (window.NodeList.prototype.forEach) {
-    return nodes.forEach(callback)
-  }
-  for (var i = 0; i < nodes.length; i++) {
-    callback.call(window, nodes[i], i, nodes)
-  }
-}
-
```

Comment:
![Animated GIF of a young Asian girl saying "The evil is defeated."](https://media.tenor.com/75EnGeBUdpgAAAAd/evil-defeated.gif)